data/json/dreams.json

```diff
@@ -449,20 +471,30 @@
     "type": "dream",
     "messages": [
       "You see yourself, five years from now, as the leader of a successful city.",
+      "Obeying orders, you revel in the satisfaction of a job well done.",
       "Your success in restoring civilization was only a dream.  For now."
     ],
     "category": "ALPHA",
     "strength": 3
   },
   {
     "type": "dream",
-    "messages": [ "NO!  You will not allow this corruption to prevail!", "You see yourself reflected in the beauty of the forest." ],
+    "messages": [
+      "NO!  You will not allow this corruption to prevail!",
+      "You see yourself reflected in the beauty of the forest.",
+      "While fearful, you dare not turn down an invitation to visit the Unseelie Court."
```

Comment:
They're mutants that are vaguely similar to literary elves. They're not actually mythological creatures.

Reply:
Yeah, I was thinking that an evolving mutant would still have dreams influenced by their human cultural legacy.  Joseph Campbell or Jungian archetypes

_"innate universal pre-conscious psychic dispositions that form the substrate from which the basic themes of human life emerge"_